packages/teleport/src/Kubes/AddKube/AddKube.tsx

```diff
@@ -0,0 +1,248 @@
+      })}
+      disableEscapeKeyDown={false}
+      onClose={onClose}
+      open={true}
+    >
+      <Flex flex="1" flexDirection="column">
+        <DialogTitle mr="auto" mb="4">
+          Add Kubernetes
+        </DialogTitle>
+        {attempt.status == 'failed' && (
+          <Alert kind="danger" children={attempt.statusText} />
+        )}
+        <DialogContent minHeight="254px" flex="0 0 auto" mb="2">
+          <Box mb={4}>
+            Install Teleport Agent in your cluster via Helm to easily connect
+            your Kubernetes cluster with Teleport. For all the available values
+            of the helm chart see{' '}
+            <Link
+              href="https://goteleport.com/docs/kubernetes-access/helm/reference/teleport-kube-agent/"
+              target="_blank"
+            >
+              the documentation
+            </Link>
+            {' .'}
+          </Box>
+          <Box mb={4}>
+            <Text>
+              <Text bold as="span">
+                Step 1
+              </Text>
+              {' - Add teleport-agent chart to your charts repository'}
+            </Text>
+            <TextSelectCopy
+              text={
+                'helm repo add teleport https://charts.releases.teleport.dev && helm repo update'
+              }
+            />
+          </Box>
+          <Box mb={4}>
+            <Text bold as="span">
+              Step 2
+            </Text>
+            {
+              ' - Generate a script to automatically configure and install the teleport-agent'
+            }
+            <Validation>
+              {({ validator }) => (
+                <Flex alignItems="center" flexDirection="row">
+                  <form
+                    onSubmit={e => handleGenerate(e, validator)}
+                    style={{ width: '100%' }}
+                  >
+                    <FieldInput
+                      mb={2}
+                      rule={(val: string) => requiredField(val, 'Namespace')}
+                      label="Namespace"
+                      autoFocus
+                      value={namespace}
+                      placeholder="teleport"
+                      width="100%"
+                      mr="3"
+                      onChange={e => setNamespace(e.target.value)}
+                    />
+                    <FieldInput
+                      mb={2}
+                      rule={(val: string) =>
+                        requiredField(val, 'Kubernetes Cluster Name')
+                      }
+                      label="Kubernetes Cluster Name"
+                      labelTip="Name shown to Teleport users connecting to the cluster."
+                      value={clusterName}
+                      placeholder="my-cluster"
+                      width="100%"
+                      mr="3"
+                      onChange={e => setClusterName(e.target.value)}
+                    />
+                    <ButtonPrimary
+                      block
+                      mt="1"
```

Comment:
the btn is too close for comfort to the input box above
```suggestion
                      mt="2"
```